test/functional/change_stream.test.js

```diff
@@ -2007,7 +2007,7 @@ describe('Change Streams', function () {
           if (counter === 2) {
             changeStream.close(close);
           } else if (counter >= 3) {
-            close(new Error('should not have received more than 2 events'));
+            close(new MongoChangeStreamError('should not have received more than 2 events'));
```

Comment:
This can also be a plain error, the errors here are all just making a test fail because they've reached a bad state, these should all probably be `expect.fail('...')` instead.